src/Symfony/Component/Form/Extension/Validator/ValidatorFormEvents.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Form\Extension\Validator;
+
+use Symfony\Component\Form\Extension\Validator\Event\PostValidateEvent;
+use Symfony\Component\Form\Extension\Validator\Event\PreValidateEvent;
+
+final class ValidatorFormEvents
+{
+    /**
+     * This event is dispatched after validation completes.
```

Comment:
fail here
also perhaps use the same doc as the class name?

Reply:
Oups, fixed.
But I do not understand what you mean by "use the same doc as the class name"

Reply:
I mean the event class doc

« This event is dispatched after the root form validation. »

Reply:
Done